ui/partials/main/thPinboardPanel.html

```diff
@@ -63,26 +63,29 @@
 
 <!-- Save UI -->
 <div id="pinboard-controls" class="btn-group-vertical"
-     title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
+     title="{{!hasPinnedJobs() ? 'No pinned jobs' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ saveUITitle('classification') }}"
+            ng-click="save()"
+            ng-disabled="!user.loggedin || !canSaveClassifications()">save
```

Comment:
!canSaveClassifications() covers the case where `!user.loggedin`, so I think you skip the check for that explciitly (both here and subsequently).